tests/tool_configurators_lvm_test.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+import os
+
+from vdsm.tool import confmeta
+
+TOPDIR = os.path.dirname(os.path.dirname(__file__))
+
+
+def test_lvmlocal_conf():
+    lvmlocal_conf = os.path.join(
+        TOPDIR, "static", "usr", "share", "vdsm", "lvmlocal.conf")
```

Comment:
This works and is good, but I would consider having the relative part of the path in a single string to make it a bit easier to read:
```python
    lvmlocal_conf = os.path.join(TOPDIR, "static/usr/share/vdsm/lvmlocal.conf")
```
Maybe even pre-assigned to a local or global variable.

But there is not much difference, so feel free to ignore it.